packages/vue/src/components/range/RangeInput.jsx

```diff
@@ -0,0 +1,280 @@
+						this.$emit('value-change', this.$data.currentValue);
+						break;
+					case 'none':
+						break;
+					default:
+						this.$emit('change', this.$data.currentValue);
+						this.$emit('valueChange', this.$data.currentValue);
+						this.$emit('value-change', this.$data.currentValue);
+						break;
+				}
+			}
+		},
+		handleOnChange(value) {
+			this.handleChange(value, 'change');
+		},
+		handleValueChange(value) {
+			this.handleChange(value, 'value-change');
+		},
+		handleInputChange(e) {
+			const { name, value } = e.target;
+			if (Number.isNaN(value)) {
+				if (name === 'start') {
+					this.$data.isStartValid = false;
+				} else {
+					this.$data.isEndValid = false;
+				}
+			} else {
+				if (name === 'start' && !this.$data.isStartValid) {
+					this.$data.isStartValid = true;
+				} else if (name === 'end' && !this.$data.isEndValid) {
+					this.$data.isEndValid = true;
+				}
+			}
+
+			if (this.$data.isStartValid && this.$data.isEndValid) {
+				if (name === 'start') {
+					this.handleChange({
+						start: Number(value),
+						end: this.$data.currentValue.end
+					});
+				} else {
+					this.handleChange({
+						start: this.$data.currentValue.start,
+						end: Number(value)
+					});
+				}
+			}
+		}
+	},
+	watch: {
+		defaultValue(newVal, oldVal) {
+			if (oldVal.start !== newVal.start || oldVal.end !== newVal.end) {
+				this.handleChange(newVal, 'none')
+			}
+		},
+		value(newVal, oldVal) {
+			if (!isEqual(newVal, oldVal)) {
+				if (this.isControlled()) {
+					this.handleChange(newVal, 'change');
+				}
+			}
+		},
+	},
+	created() {
+		if (this.$props.defaultValue && this.$props.defaultValue.start && this.$props.defaultValue.end) {
+			this.handleChange(this.$props.defaultValue, 'none')
+		}
+	},
+	render() {
+		const {
+			className,
+			dataField,
+			range,
+			rangeLabels,
+			componentId,
+			innerClass,
+			stepValue,
+			componentStyle,
+			themePreset,
+			includeNullValues,
+			beforeValueChange,
+			customQuery,
+			data,
+			filterLabel,
+			react,
+			showFilter,
+			showCheckbox,
+			title,
+			URLParams,
+			sliderOptions,
+			nestedField,
+		} = this.$props;
+		return (
+			<Container style={componentStyle} class={className}>
+				<RangeSlider
+					componentId={componentId}
+					value={{
+						start: this.$data.currentValue.start,
+						end: this.$data.currentValue.end
+					}}
+					range={range}
+					dataField={dataField}
+					rangeLabels={rangeLabels}
+					includeNullValues={includeNullValues}
+					beforeValueChange={beforeValueChange}
+					customQuery={customQuery}
+					data={data}
+					filterLabel={filterLabel}
+					react={react}
+					showFilter={showFilter}
+					showCheckbox={showCheckbox}
+					title={title}
+					URLParams={URLParams}
+					sliderOptions={sliderOptions}
+					nestedField={nestedField}
+					on-change={this.handleOnChange}
+					on-value-change={this.handleValueChange}
+				/>
+				<Flex className={getClassName(innerClass, 'input-container') || null}>
```

Comment:
```suggestion
				<Flex class={getClassName(innerClass, 'input-container') || null}>
```
`className` doesn't work for vue.